MAUI/NumericEntry/Getting-Started.md

```diff
@@ -0,0 +1,266 @@
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfNumericEntry HorizontalOptions="Center"
+                     VerticalOptions="Center"
+                     Value="12315"
+                     AllowNull="False" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfNumericEntry sfNumericEntry = new SfNumericEntry();
+sfNumericEntry.HorizontalOptions = LayoutOptions.Center;
+sfNumericEntry.VerticalOptions = LayoutOptions.Center;
+sfNumericEntry.AllowNull = false;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI NumericEntry prevent empty textbox](GettingStarted_images/allowNull.gif)
+
+## Setting Placeholder Text
+
+You can prompt the user with any information by using the `PlaceholderText` property. Watermark text will be displayed only when the value of the `AllowNull` property is **true** and the value of `NumericEntry` control is **null**. The default value of `Placeholder` property is **string.Empty** (No string will be displayed).
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfNumberBox HorizontalOptions="Center" 
+                     VerticalOptions="Center" 
+                     Placeholder="Enter input here..." />
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfNumberBox SfNumberBox= new SfNumberBox();
+SfNumberBox.Placeholder = "Enter input here...";
+sfNumberBox.HorizontalOptions = LayoutOptions.Center;
+sfNumberBox.VerticalOptions = LayoutOptions.Center;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI NumericEntry Placeholder Text](GettingStarted_images/placeholder_text.png)
+
+## Clear button visibility 
+
+The `ShowClearButton` property is used to show or hide the clear button in `NumericEntry`. By default, visibility of the clear button is enabled.
+
+N> The clear button appears only when the text box is focused and the `IsEditable` property value is set to **true**.
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfNumericEntry x:Name="numericEntry"  
+            WidthRequest="300"                   
+            ShowClearButton="True" 
+            IsEditable="True" 
+            Value="10"/>
+
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfNumericEntry SfNumericEntry = new SfNumericEntry();
+SfNumericEntry.ShowClearButton = true;
+SfNumericEntry.IsEditable = true; 
+
+
+{% endhighlight %}
+{% endtabs %}
+
+If IsEditable is true
+
+![.NET MAUI NumericEntry with Clear Button](GettingStarted_images/clearbutton_visible.png)
+
+If IsEditable is false
+
+![.NET MAUI NumericEntry without Clear Button](GettingStarted_images/clearbutton_collapsed.png)
+
+
+## Value changed notification
+
+The `ValueChanged` event is triggered, when the `Value` property of `NumericEntry` control is changed. The value will not be changed when the user enters the input. The value of the `NumericEntry` control will be changed after validation is performed on the `Enter` keypress or when the focus is lost in the control. The `ValueChanged` contains the following properties.
```

Comment:
The `ValueChanged` event is triggered when the `Value` property of the `NumericEntry` control is changed. The value will not be changed when the user enters the input. The value of the `NumericEntry` control will be changed after validation is performed on the `Enter` keypress or when the focus is lost in the control. The `ValueChanged` contains the following properties.

Reply:
The requested content modification has been successfully addressed